.github/workflows/test-private.yml

```diff
@@ -9,39 +13,28 @@ on:
     branches:
       - "main"
       - "master"
```

Comment:
lets make this more representative, by making it like this: https://github.com/balena-os/balena-intel/pull/756/files#diff-e0cd94e097ab180ba23b4b957776c71a00c7458cc6141e27ff6bb8f1fff2e94fR10

Reply:
done